multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.120.0-core.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="sap.com" id="alter_operation_table_add_not_null_constraint_current_state">
+        <addNotNullConstraint tableName="operation"
```

Comment:
Can we verify that there it doesn't exists any null values?

Reply:
Not really sure how this can be done. Back when we had a log when the calculated and the cached state differed, there were nulls only at one point in time - obviously when the new version was applied. So I suppose that there are no nulls.